src/main/java/seedu/address/ui/OrderDisplay.java

```diff
@@ -44,5 +46,11 @@ public OrderDisplay(Order order) {
         orderNumber.setText(order.getOrderNumber().toString());
         orderPerson.setText(order.getPerson().getName().fullName);
         orderDetails.setText(StringUtil.unmodifiableSetToCommaSeparatedStr(order.getMedicines()));
+        status.setText(order.getStatus().toString());
+        status.setWrapText(true);
```

Comment:
is this necessary here?

Reply:
that one is used to change the colour depending on the status